### ISSUE TITLE
window is not defined for cookie consent banner

### ISSUE THREAD

**Issue by bryyyanribo** [ambitiondev/cookiebot#9]
I have followed the documentation implementing the Cookiebot in my Nuxt3 application. When I run it, I'm getting an error message saying window is not defined. I have a if condition to check if it's undefined before the cookie gets loaded but doesn't seem to work. Is there any work around this?

```
App.vue
<script setup lang="ts">
const cookieDeclarationRef = ref<HTMLElement | null>(null);

const { consentBanner } = useCookiebot({
  // options
});

onMounted(() => {
  if (process.client && typeof window !== undefined) { 
    consentBanner();
  }
});
</script>

<template>
  <nuxt-layout>
    <nuxt-page />
  </nuxt-layout>
</template>
```

```

Nuxt Config:
  modules: [
    "@ambitiondev/nuxt-cookiebot",
  ],
  cookiebot: {
    cookieBotId: "ID_HERE",
  },

```

**Comment by bnachtweh** [comment]
> I have followed the documentation implementing the Cookiebot in my Nuxt3 application. When I run it, I'm getting an error message saying window is not defined. I have a if condition to check if it's undefined before the cookie gets loaded but doesn't seem to work. Is there any work around this?
> 
> ```
> App.vue
> <script setup lang="ts">
> const cookieDeclarationRef = ref<HTMLElement | null>(null);
> 
> const { consentBanner } = useCookiebot({
>   // options
> });
> 
> onMounted(() => {
>   if (process.client && typeof window !== undefined) { 
>     consentBanner();
>   }
> });
> </script>
> 
> <template>
>   <nuxt-layout>
>     <nuxt-page />
>   </nuxt-layout>
> </template>
> ```
> 
> ```
> 
> Nuxt Config:
>   modules: [
>     "@ambitiondev/nuxt-cookiebot",
>   ],
>   cookiebot: {
>     cookieBotId: "ID_HERE",
>   },
> ```

Can you provide a minimal reproduction project and/or description of the stack you're using?

**Comment by bryyyanribo** [comment]
I'm using nuxt version: "nuxt": "^3.8.1",

**Comment by bnachtweh** [comment]
@bryyyanribo your issue should have been resolved in version 3.1.0. You can install it by running `npm i @ambitiondev/nuxt-cookiebot@latest`
